Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CardNumberView.swift
+//  Frames
+//
+//  Created by Harry Brown on 05/07/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+protocol CardNumberViewProtocol: AnyObject {
+  var schemeIcon: Constants.Bundle.SchemeIcon { get set }
+}
+
+final class CardNumberView: UIView, CardNumberViewProtocol {
```

Comment:
I am assuming that `CardNumberView` will be used by merchant developers this should be public ? if so then can you also update the API Spec.

Reply:
Have left as internal for now